odl/contrib/fom/supervised.py

```diff
@@ -538,9 +525,12 @@ def psnr(data, ground_truth, normalized=False):
         Input data to compare to the ground truth.
     ground_truth : `FnBaseVector`
         Reference to compare ``data`` to.
-    normalized : bool
-        If true, normalize ``data`` and ``ground_truth`` to have the
-        same mean and variance before comparison.
+    use_zscore : bool
```

Comment:
Why this change? It breaks uniformity of the API.

Reply:
Basically the normalization here is of the *input* while the others normalize their *output*. Having the same name was quite bad programming (in my opinion)

Reply:
Good point. So the `normalized` option is simply missing.

Reply:
PSNR cannot in any "nice" way be normalized. E.g. it can go from -infty to infty, and irritatingly, infty is the value we get from `psnr(x, x)` so normalizing it is "weird". Sure, we could apply something like e.g. a sigmoid, but well, what would the end really mean then? But perhaps its better than nothing.

Reply:
Hm, so how about this: If `M` is the squared max of the ground truth, and `E` the mean squared error, then
```
PSNR = log_10( (M / E)^(10) ) = ln( (M / E)^(10) ) / ln(10)
```
So putting that into a sigmoid `s(x) = 1 / (1 + e^(-a x))` for `a = ln(10)` means
```
s(PSNR) = 1 / (1 + e^(-ln( (M/E)^(10) )) = 1 / (1 + (M/E)^(-10))
```
which would be very close to 1 for anything relevant. But by choosing a smaller `a` one could find something that yields 0.5 for "moderate" PSNR and close to 1 for "high" PSNR, where those are experience values. Maybe moderate would be 20 or so, and high 40?

Reply:
Well we could let users simply select the parameter. I'd probably write the normalized formula

```
nPSNR(x) = 1 / (1 + e^(-x/a))
```

and then users can easily state a "standard error", e.g. 20 decibels.

Reply:
Sure, why not. It needs a bit of experimenting I think, but some normalization for a user-specified db value would make sense.